site/docs/v1/tech/apis/_form-request-body.adoc

```diff
@@ -45,6 +45,7 @@ The type of form being created, a form type cannot be changed after the form has
  * `registration` - This form will be used for self service registration.
  * `adminRegistration` - This form be used to customize the add and edit User Registration form in the FusionAuth UI. [since]#Available since 1.20.0#
```

Comment:
Should these be in alphabetical order?

Reply:
They are in enum order, I think.  Probably good to reorder.